omnibus/config/projects/chefdk.rb

```diff
@@ -32,6 +32,11 @@
   install_dir "#{default_root}/#{name}"
 end
 
+override :ruby, version: "2.1.8"
+override :'ruby-windows-devkit', version: "4.7.2-20130224" if windows? && windows_arch_i386?
```

Comment:
No need for any override...  ruby-windows-devkit will pull in the most recent version for each architecture correctly.